rDNA/R/rDNA.R

```diff
@@ -1844,6 +1847,21 @@ print.dna_barplot <- function(x, trim = 30, attr = TRUE, ...) {
 #' 
 #' # plot barplot with ggplot2                
 #' autoplot(b)
+#' 
+#' ## Change barplot colors to other colors than specified in color attribute
+#' 
+#' # compute barplot data for organizations
+#' b <- dna_barplot(statementType = "DNA Statement",
+#'                  variable = "organization",
+#'                  qualifier = "agreement")
+#'                  
+#' # Change color for NGOs to red
+#' b$Color[b$Type == "NGO"] <- "red"
+#' 
+#' # Change color for Government to blue
+#' b$Color[b$Type == "Government"] <- "blue"
+#' 
+#' autoplot(b, colors = TRUE)
```

Comment:
Good example. Can you also add an example where the user uses an attribute variable column, for example "type", to add colours to the plot via ggplot2?